app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/CustomPlaybackTransportControlGlue.java

```diff
@@ -59,6 +70,67 @@
         initActions(context);
     }
 
+    @Override
+    protected PlaybackRowPresenter onCreateRowPresenter() {
+        final AbstractDetailsDescriptionPresenter detailsPresenter =
+                new AbstractDetailsDescriptionPresenter() {
+                    @Override
+                    protected void onBindDescription(ViewHolder
+                                                             viewHolder, Object obj) {
+                        PlaybackTransportControlGlue glue = (PlaybackTransportControlGlue) obj;
+                        viewHolder.getTitle().setText(glue.getTitle());
+                        viewHolder.getSubtitle().setText(glue.getSubtitle());
+                    }
+                };
+
+        PlaybackTransportRowPresenter rowPresenter = new PlaybackTransportRowPresenter() {
+            @Override
+            protected RowPresenter.ViewHolder createRowViewHolder(ViewGroup parent) {
+                RowPresenter.ViewHolder vh = super.createRowViewHolder(parent);
+                Context context = getContext();
+
+                if (context != null) {
+                    mEndsText = new TextView(context);
+                    mEndsText.setTextAppearance(context, R.style.PlaybackControlsText);
```

Comment:
Can we use `playbackControlsTimeStyle` here?

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:leanback/leanback/src/main/res/layout/lb_playback_transport_controls_row.xml;l=113

Reply:
Apparently, yes. I did not know that. Although it's not using the attr but instead the style the attr value points to.